[docs-only] add auth bearer readme (#4982)
owncloud/ocis: services/auth-basic/README.md

# Auth-Basic Service

> The oCIS Auth Basic service provides basic authentication for those clients who cannot handle OpenID Connect. This is a rare case, is usually not necessary and mainly used for tests or development.

Reviewer: ```suggestion
The oCIS Auth Basic service provides basic authentication for those clients who cannot handle OpenID Connect. This should only be enabled for tests and development.
```
